src/Config/Validator/Deploy/DatabaseSplitConnection.php

```diff
@@ -63,15 +63,20 @@ public function validate(): Validator\ResultInterface
             return $this->resultFactory->success();
         }
 
-        return $this->resultFactory->error(sprintf(
-            'Split database configuration was detected in the property %s'
-            . ' of the file .magento.env.yaml:' . PHP_EOL
-            . '%s' . PHP_EOL
-            . 'Magento Cloud does not support a custom split database configuration,'
-            . ' such configurations will be ignored',
-            DeployInterface::VAR_DATABASE_CONFIGURATION,
-            implode(PHP_EOL, $messageItem),
+        return $this->resultFactory->error(
+            sprintf(
+                'Detected split database configuration in the %s property of the file .magento.env.yaml:' . PHP_EOL
+                . '%s' . PHP_EOL
+                . 'Magento Cloud does not support a custom split database configuration,'
+                . ' such configurations will be ignored',
+                DeployInterface::VAR_DATABASE_CONFIGURATION,
+                implode(PHP_EOL, $messageItem)
+            ),
+            sprintf(
+                'Remove custom connections for split databases from the %s variable in .magento.env.yaml',
```

Comment:
```suggestion
                'Update the %s variable in the .magento.env.yaml file to remove custom connections from the split database configuration.',
```

Aren't the only valid connections "checkout" and "sales". Should the error message be more specific? For example:  Magento Cloud supports split database connections for `checkout` and `sales` databases only?